ViewModel/Inventory/DefineSIAttributesVM.cs

```diff
@@ -237,17 +238,17 @@ private List<StructureMissingDataRowItem> GetMissingTerrainElevations()
             int badElevationNumber = -9999;
 
             StructureElevationsFromTerrainFile elevsFromTerrainHelper = new StructureElevationsFromTerrainFile(_Path);
-            float[] elevs = elevsFromTerrainHelper.GetStructureElevationsFromTerrainFile(ref errorMessage);
+            _StructureElevations = elevsFromTerrainHelper.GetStructureElevationsFromTerrainFile(ref errorMessage).ToList();
```

Comment:
i made some changes in "GetStructureElevationsFromTerrainFile" so that it now returns a list. I don't know what can happen from the "GridReader" call so i do a null check there. If nothing comes back or it is null then i add an error message.